Sources/Components/SelectorTitleView/SelectorTitleView.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol SelectorTitleViewDelegate: AnyObject {
+    func selectorTitleViewDidSelectButton(_ view: SelectorTitleView)
+}
+
+public class SelectorTitleView: UIView {
+    public enum ArrowDirection {
+        case up
+        case down
+    }
+
+    public weak var delegate: SelectorTitleViewDelegate?
+
+    public var arrowDirection: ArrowDirection = .down {
+        didSet {
+            let asset: FinniversImageAsset = arrowDirection == .up ? .arrowUpSmall : .arrowDownSmall
+            button.setImage(UIImage(named: asset), for: .normal)
+        }
+    }
+
+    private var isEnabled: Bool = true {
+        didSet {
+            button.isEnabled = isEnabled
+        }
+    }
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel(withAutoLayout: true)
+        label.font = UIFont.captionStrong.withSize(12).scaledFont(forTextStyle: .footnote)
+        label.adjustsFontForContentSizeCategory = true
+        label.textAlignment = .center
+        label.text = heading
+        label.textColor = .textSecondary
+        return label
+    }()
+
+    private lazy var button: UIButton = {
+        let button = UIButton(withAutoLayout: true)
+        button.titleLabel?.font = UIFont.bodyStrong.withSize(17).scaledFont(forTextStyle: .footnote)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+
+        let spacing = .smallSpacing / 2
+
+        button.semanticContentAttribute = .forceRightToLeft
+        button.imageEdgeInsets = UIEdgeInsets(top: spacing, leading: spacing, bottom: 0, trailing: -spacing)
+        button.titleEdgeInsets = UIEdgeInsets(top: 0, leading: -spacing, bottom: 0, trailing: spacing)
+        button.addTarget(self, action: #selector(handleButtonTap), for: .touchUpInside)
+
+        let buttonColor: UIColor = .btnPrimary
```

Comment:
Could this be a property with a default value instead? For the tableHeaderView in favorites this needs to have a different color

Reply:
Made it into a method so it's possible to set the disabled color as well.